lib/matplotlib/figure.py

```diff
@@ -3003,7 +3003,12 @@ def _set_base_canvas(self):
         This is used upon initialization of the Figure, but also
         to reset the canvas when decoupling from pyplot.
         """
+        # check if we have changed the API due to hi-dpi screens
```

Comment:
```suggestion
        # check if we have changed the DPI due to hi-dpi screens
```

Reply:
well, that is an aggressive Freudian slip.

Reply:
I would hate to have to remember a different API depending which monitor I'm attached to...